### ISSUE TITLE
Feature suggestion: Import canisters created from an actor class

### ISSUE THREAD

**Issue by vporton** [dfinity/motoko#4206]
Actor class:
```motoko
shared ({ caller = _owner }) actor class PST(
    creator : {owner : Principal; subaccount : ?ICRC1Types.Subaccount},
) : async ICRC1.FullInterface {
    ...
}
```
I create its instance by `dfx deploy zon_pst --argument 'record { owner = principal "racnx-sccpy-mgfgr-rgb67-bvwyx-gjkad-lyw33-prq23-yw24r-eb65i-oqe"; subaccount = null; }'` (`zon_pst` is its identifier in `dfx.json`).

After this `import PST "canister:zon_pst";` should work, but it produces `import error [M0011], canister alias "zon_pst" not defined`. (That is, importing an instance of an actor class is not supported. I recommend to add its support.)

**Comment by crusso** [comment]
Yes, dfx isn't very good at handling Motoko's actor classes, although I thought it did (at least at one time) let you do the scenario above. Maybe it only works if the argument to the class is trivial ('()'). Or maybe it never worked at all.

It's not clear what it would mean though. Does deploying the class just deploy one instance, and where does argument come from, if not from the command line, then maybe from dfx.json. Should re-deployment update the existing instance, or create a new one (I expect the former).

**Comment by vporton** [comment]
> It's not clear what it would mean though. Does deploying the class just deploy one instance, and where does argument come from, if not from the command line, then maybe from dfx.json. Should re-deployment update the existing instance, or create a new one (I expect the former).

Deploying a class just deploys one instance in the current DFX, I don't call to change this.

If it was not deployed from the command line, just don't deploy the class at all.

Re-deployment currently updates the existing instance, I don't call to change this.
